packages/fuels-core/src/codec/abi_decoder/bounded_decoder.rs

```diff
@@ -36,7 +35,7 @@ impl BoundedDecoder {
         }
     }
 
-    pub fn decode(&mut self, param_type: &ParamType, bytes: &[u8]) -> Result<Token> {
+    pub(crate) fn decode(&mut self, param_type: &ParamType, bytes: &[u8]) -> Result<Token> {
```

Comment:
Why are we making it public only to this crate? I'm not sure I recall this correctly, but I thought some people were using this function directly, e.g.: indexer. If that's not the case then it's all good

Reply:
I think this was just a lapsus. `BoundedDecoder` is `pub(crate)` and not exposed so this function would not be reachable.